megamek/src/megamek/client/ui/swing/MovementDisplay.java

```diff
@@ -4096,6 +4096,13 @@ public void gameTurnChange(GameTurnChangeEvent e) {
         // if all our entities are actually done, don't start up the turn.
         if (clientgui.getClient().getGame().getPlayerEntities(clientgui.getClient().getLocalPlayer(), false)
                 .stream().allMatch(Entity::isDone)) {
+            String playerName;
```

Comment:
This is a direct copy of the code 20 lines below. It could be moved to a method:

```
private void setStatusBarTextOthersTurn(@Nullable Player player) {
    String playerName = (player != null) ? player.getName() : "Unknown";
    setStatusBarText(Messages.getString("MovementDisplay.its_others_turn", playerName));
}
```

and then called as setStatusBarTextOthersTurn(e.getPlayer());

This would look a little neater.